src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -504,6 +508,7 @@ struct program_node {
     bool data_flow = false;
     bool in_shape_of_subgraph = false;
     bool runtime_skippable = false;
+    uint8_t consecutive_runtime_skippable_count = 0;
```

Comment:
As I mentioned offline, please don't add a member to node. The information is only used by the pass, so it would be enough to add this in the pass.

Reply:
Updated it as commented. Thanks!